docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -268,12 +269,23 @@ Note that only one reporter can write to `stdout`. All other reporters must outp
 
 ### -s \<path\>, --screenshots \<path\>
 
-Enables screenshot capturing and specifies the directory where screenshots are saved.
+Enables screenshot capturing and specifies the root directory where screenshots are saved.
```

Comment:
Enables screenshot capturing and specifies the root directory where screenshots are saved -> Enables screenshots and specifies the root directory where they are saved